crosshair/__init__.py

```diff
@@ -1,7 +1,16 @@
+"""Statically analyze Python code using symbolic execution."""
+
 from crosshair.core import realize
 from crosshair.core import with_realized_args
 from crosshair.core import register_patch
 from crosshair.core import register_type
 from crosshair.statespace import StateSpace
 from crosshair.util import IgnoreAttempt
 from crosshair.util import debug
+
+# Do not forget to update in setup.py!
+__version__ = "0.0.9"
+__author__ = "Phillip Schanely"
+__copyright__ = "Copyright 2019-2021 Phillip Schanely"
```

Comment:
Please double-check the copyright! It will also be used in Sphinx in the following pull request.

Reply:
Hmm. My vague impression is that by default copyright is held per-contribution. Can we leave it out, or say "varies" or something?